SDC/data-generation.js

```diff
@@ -0,0 +1,168 @@
+  { id: '048', ticker: 'DBX', company:  'Dropbox'},
+  { id: '049', ticker: 'WMT', company:  'Walmart'},
+  { id: '050', ticker: 'JCP', company:  'J.C. Penney'},
+  { id: '051', ticker: 'GM', company:   'GM'},
+  { id: '052', ticker: 'VTI', company:  'Vanguard Total'},
+  { id: '053', ticker: 'BILI', company:     'Bilibili'},
+  { id: '054', ticker: 'NOK', company:  'Nokia'},
+  { id: '055', ticker: 'GLUU', company:     'Glu Mobile'},
+  { id: '056', ticker: 'VZ', company:   'Verizon'},
+  { id: '057', ticker: 'VSLR', company:     'Vivint Solar'},
+  { id: '058', ticker: 'SHOP', company:     'Shopify'},
+  { id: '059', ticker: 'CARA', company:     'Cara Therapeutics'},
+  { id: '060', ticker: 'SNE', company:  'Sony'},
+  { id: '061', ticker: 'PFE', company:  'Pfizer'},
+  { id: '062', ticker: 'ENPH', company:     'Enphase Energy'},
+  { id: '063', ticker: 'CVS', company:  'CVS'},
+  { id: '064', ticker: 'SPOT', company:     'Spotify'},
+  { id: '065', ticker: 'COST', company:     'Costco'},
+  { id: '066', ticker: 'TRXC', company:     'TransEnterix'},
+  { id: '067', ticker: 'TWLO', company:     'Twilio'},
+  { id: '068', ticker: 'PCG', company:  'PG&E'},
+  { id: '069', ticker: 'KHC', company:  'Kraft Foods'},
+  { id: '070', ticker: 'INSY', company: 'Insys Therapeutics'},
+  { id: '071', ticker: 'AKS', company:  'AK Steel'},
+  { id: '072', ticker: 'LUV', company:  'Southwest Airlines'},
+  { id: '073', ticker: 'CRSP', company: 'CRISPR'},
+  { id: '074', ticker: 'FDX', company:  'FeDex'},
+  { id: '075', ticker: 'VKTX', company: 'Viking Therapeutics'},
+  { id: '076', ticker: 'JPM', company:  'JPMorgan Chase'},
+  { id: '077', ticker: 'DNR', company:  'Denbury'},
+  { id: '078', ticker: 'SPWR', company: 'SunPower'},
+  { id: '079', ticker: 'UAA', company:  'Under Armour'},
+  { id: '080', ticker: 'BOTZ', company: 'Global X Robotics'},
+  { id: '081', ticker: 'SFIX', company: 'Stitch Fix'},
+  { id: '082', ticker: 'AMAT', company: 'Applied Materials'},
+  { id: '083', ticker: 'YETI', company: 'YETI'},
+  { id: '084', ticker: 'EA', company:   'EA'},
+  { id: '085', ticker: 'QCOM', company: 'Qualcomm'},
+  { id: '086', ticker: 'TGT', company:  'Target'},
+  { id: '087', ticker: 'TEVA', company: 'Teva Pharmaceutical'},
+  { id: '088', ticker: 'JNJ', company:  'Johnson & Johnson'},
+  { id: '089', ticker: 'IIPR', company: 'Innovative Industrial Properties'},
+  { id: '090', ticker: 'ACB', company:  'Aurora Cannabis'},
+  { id: '091', ticker: 'GE', company:   'GE' },
+  { id: '092', ticker: 'AAPL', company: 'Apple' },
+  { id: '093', ticker: 'F', company:    'Ford' },
+  { id: '094', ticker: 'CRON', company: 'Cronos Group'},
+  { id: '095', ticker: 'MSFT', company: 'Microsoft' },
+  { id: '096', ticker: 'GPRO', company: 'GoPro' },
+  { id: '097', ticker: 'FIT', company:  'Fitbit' },
+  { id: '098', ticker: 'AMD', company:  'AMD'},
+  { id: '099', ticker: 'FB', company:   'Facebook' },
+  { id: '100', ticker: 'CGC', company:  'Canopy Growth'}
+];
```

Comment:
Looks good- for small improvements in lookups, you could consider a Set rather than an Array data structure.